Earthfile

```diff
@@ -2,33 +2,70 @@
 
 VERSION 0.8
 
+alpine-with-nix:
+  FROM alpine:20240329
+  # need the 'testing'-repo to install `nix`
+  RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+  RUN apk add --no-cache nix bash
+  RUN mkdir -p /etc/nix && echo "extra-experimental-features = nix-command flakes" >> /etc/nix/nix.conf
+  # replace /bin/sh with a script that sources `/root/sh_env` for every RUN command.
+  # we use this to execute all `RUN`-commands in our nix dev shell.
+  # need to explicitly delete `/bin/sh` first, because it's a symlink to `/bin/busybox`,
+  # and `COPY` would actually follow the symlink and replace `/bin/busybox` instead.
```

Comment:
```suggestion
  # replace /bin/sh with a script that sources `/root/sh_env` for every RUN command.
  # we use this to execute all `RUN`-commands in our nix dev shell.
  # we need to explicitly delete `/bin/sh` first, because it's a symlink to `/bin/busybox`,
  # and `COPY` would actually follow the symlink and replace `/bin/busybox` instead.
```